powerauth-java-http/src/main/java/io/getlime/security/powerauth/http/PowerAuthSignatureHttpHeader.java

```diff
@@ -51,6 +51,11 @@ public static class Key {
          */
         private static final String SIGNATURE_TYPE = "pa_signature_type";
 
+        /**
+         * Key representing the "pa_temporary_key_id" in the PowerAuth authorization header.
+         */
+        private static final String TEMPORARY_KEY_ID = "pa_temporary_key_id";
+
```

Comment:
This is no longer a option. The mobile client will send a separate encryption header if both, encryption and PA signature is used.